staging/src/k8s.io/apimachinery/pkg/util/httpstream/spdy/roundtripper.go

```diff
@@ -158,27 +158,28 @@ func (s *SpdyRoundTripper) dial(req *http.Request) (net.Conn, error) {
 		return nil, err
 	}
 
```

Comment:
this bit fixes two existing bugs:
* mutation of the roundtripper
* mutation of the original tlsConfig (which the round tripper should not assume it is free to modify)